rocketc/rocketc.py

```diff
@@ -446,4 +454,57 @@ def _remove_user_from_group(self, group_name, user_id):
         group = api.search_rocket_chat_group(group_name)
         if group["success"]:
             group = group["group"]
-            api.kick_user_from_group(user_id, group["_id"])
+            return api.kick_user_from_group(user_id, group["_id"])
+        return group
+
+    def get_team_members(self, username):
```

Comment:
If I'm not wrong, there was a get_team_members in the code, why does it appear here as a new added method? did you delete it and here you're adding it again?

Reply:
you are talking about other xblock